_security-analytics/api-tools/detector-api.md

```diff
@@ -7,60 +7,58 @@ nav_order: 35
+`name` | String | Name of the detector. Name should only consist of upper and lowercase letters, numbers 0-9, hyphens, spaces, and underscores. Use between 5 and 50 characters. Required.
+`detector_type` | String | The log type that defines the detector. Options are `linux`, `network` ,`windows`, `ad_ldap`, `apache_access`, `cloudtrail`, `dns`, and `s3`. Required.
+`schedule` | Object | The schedule that determines how often the detector runs. For information on specifying fixed intervals in the API, see [Cron expression reference]({{site.url}}{{site.baseurl}}/monitoring-plugins/alerting/cron/).
+`schedule.period` | Object | Details for the frequency of the schedule.
+`schedule.period.interval` | Integer | The interval at which the detector runs.
+`schedule.period.unit` | String | The interval's unit of time.
+`inputs` | Object | Detector inputs for its definition.
+`inputs.detector_input` | Array | An array that contains the index and definition used to create the detector. For the initial release in 2.4, only one log data source is allowed for the detector. Support for associating multiple log data sources with a single detector will be available in a future release. See description for `inputs.detector_input.indices` and the log data source.
+`inputs.detector_input.description` | String | Description of the detector. Optional.
+`inputs.detector_input.custom_rules` | Array | Detector inputs for custom rules. At least one rule must be specified for a detector. Optional if pre-packaged rules are specified.
+`inputs.detector_input.custom_rules.id` | String | A valid rule ID for the custom rule generated by the user. Valid rules are formatted as a globally, or, universally unique identifier (UUID) See [Universally unique identifier](https://en.wikipedia.org/wiki/Universally_unique_identifier) for more information.
+`inputs.detector_input.indices` | String | The log data source used for the detector. Required.
+`inputs.detector_input.pre_packaged_rules` | Array | Detector inputs for pre-packaged rules (as opposed to custom rules). At least one rule must be specified for a detector. Optional if custom rules are specified.
+`inputs.detector_input.pre_packaged_rules.id` | String | The rule ID for pre-packaged rules. See [Search Pre-Packaged Rules]({{site.url}}{{site.baseurl}}/security-analytics/api-tools/rule-api/#search-pre-packaged-rules) for information on how to use the API to search for rules and obtain rule IDs in results.
+`triggers` | Array | Trigger settings for alerts.
+`triggers.ids` | Array | A list of rule IDs that become part of the trigger condition.
+`triggers.tags` | Array | Tags are specified in a security rule. Tags can then be selected and applied to the alert trigger to focus the trigger conditions for alerts. See an example of how tags are used in a Sigma rule in Sigma's [Rule Creation Guide](https://github.com/SigmaHQ/sigma/wiki/Rule-Creation-Guide#tags).
+`triggers.id` | String | The unique ID for the trigger. User generated.
```

Comment:
@sbcd90 is this user generated?

Reply:
@cwillum I would remove the `User generated` piece here because we don't pass it from the frontend